src/ui/Onboarding/Import/SelectWallets.tsx

```diff
@@ -38,12 +39,19 @@ export function SelectWallets({
     suspense: false,
   });
 
-  const { value: activeWallets, isLoading: activeWalletsAreLoading } =
+  const activeWalletsRequestTimeout = useRenderDelay(10000);
```

Comment:
Why not use `useStaleTime`? https://github.com/zeriontech/zerion-wallet-extension/blob/dcee6dae5e5aca2a9731f799dfba3ed0c1e068b8/src/ui/pages/GetStarted/ImportWallet/MnemonicImportView/MnemonicImportView.tsx#L82-L83